precompiles/conviction-voting/ConvictionVoting.sol

```diff
@@ -33,6 +33,90 @@ interface ConvictionVoting {
         Locked6x
     }
 
+    struct ClassLock {
```

Comment:
Please add standard documentation for all the structs

Reply:
I think the standard for struct param is to use 
`/// @param xxx description`

Reply:
Hmm our other contracts also use the same format. I also don't see anything specific for structs here. `@param` is only for function and events.
 https://docs.soliditylang.org/en/v0.8.17/natspec-format.html